nodejs/week1/meals/src/backend/data/reservations.json

```diff
@@ -0,0 +1,27 @@
+[
+    {
+      "id": 1,
+      "name": "Benjamin Hughes",
+      "email": "benjamin@hughes.dk",
+      "mealId": 1
+    },
+    {
+        "id": 2,
+        "name": "Anna John",
+        "email": "ajgcph@123.dk",
+        "mealId": 2
+      },
+      {
+        "id": 3,
+        "name": "Evan Cherian",
+        "email": "evan@cherian.dk",
+        "mealId": 1
+      },
+      {
+        "id": 4,
+        "name": "Jayden Varghese",
+        "email": "jayden@varghese.dk",
+        "mealId": 2
+      }
+
+  ]
```

Comment:
Fix formatting. Keep indentation consistent in this file.

Tip: Set up your text editor with something like prettier. It will automatically format your code everytime you save the file so you don't need to even think about it.

For example, VSCode has this plugin https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode